service/templates/patches.html

```diff
@@ -28,22 +28,25 @@ <h1>
       Error loading data from server: [[patchesError]]
     </div>
 
-    <div  ng-if="patches.length > 0"
-          class="btn-group btn-group-sm pagination-fixed">
-      <span ng-if="currentPage > 0">
-        <a  class="btn btn-default"
-            style="cursor: pointer"
-            ng-click="previousPage()">
-          Newer
-        </a>
-      </span>
-      <span>
-        <a  class="btn btn-default"
-            style="cursor: pointer"
-            ng-click="nextPage()">
-          Older
-        </a>
-      </span>
+    <div class="form-inline" ng-if="patches.length > 0">
```

Comment:
in the (albeit unlikely) scenario where filtering the patches results in 0 patches will there be a way to remove the filter?

Reply:
Ah weird edge case. I'll move the ng-if down past the checkbox.

Reply:
Actually I noticed this happened when I scrolled to the end of my patches in my local evergreen too, so I'll just get rid of it overall.